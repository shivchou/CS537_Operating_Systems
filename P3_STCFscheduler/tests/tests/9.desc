Testing exec2() system call with invalid completion_time
