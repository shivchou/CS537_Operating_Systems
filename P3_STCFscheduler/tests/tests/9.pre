./edit-makefile.sh ../solution/Makefile test_9 > ../solution/Makefile.test
cp -f tests/test_9.c ../solution/test_9.c
cd ../solution/
make clean
make -f Makefile.test clean
make -f Makefile.test xv6.img
make -f Makefile.test fs.img
cd ../tests
