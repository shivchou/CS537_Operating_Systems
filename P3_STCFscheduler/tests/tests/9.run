cd ../solution; ../tests/run-xv6-command.exp CPUS=1 SCHEDULER=STCF Makefile.test test_9 | grep -E 'XV6_TEST'; cd ../tests
