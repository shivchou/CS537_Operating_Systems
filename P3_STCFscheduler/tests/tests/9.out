XV6_TEST: scheduler : STCF
XV6_TEST: exec2() test started
XV6_TEST: exec2(0) returned -1
XV6_TEST: exec2(-50) returned -1
XV6_TEST: pid 4 started
XV6_TEST: pid 4 finished
XV6_TEST: exec2() test finished
